manage.py

```diff
@@ -413,8 +541,213 @@ def test_container_key_value():
     check_container_keys_vakues()
 
 
+@manager.command
+@manager.option(
+    "-d",
+    "--datasource",
+    help="data source name, indexing all the data sources is the default",  # noqa
+)
+@manager.option(
+    "-f",
+    "--folder",
+    help="data folder which contains CSV files",  # noqa
+)
+@manager.option(
+    "-r",
+    "--resource",
+    help="resource name, creating all the indexes for all the resources is the default",  # noqa
+)
+@manager.option(
+    "-n",
+    "--need_convert",
+    help="if CSV files are generating from CSV templates, this attribute needs to be true",  # noqa
+)
+@manager.option("-u", "--update_cache", help="update the cache")
+def get_index_data_from_csv_files(
+    datasource=None,
+    folder=None,
+    resource="image",
+    need_convert="False",
+    update_cache="False",
+):
+    from omero_search_engine.cache_functions.elasticsearch.transform_data import (
+        insert_resource_data,
+        # save_key_value_buckets,
+    )
+    import json
+
+    update_cache = json.loads(update_cache.lower())
+    need_convert = json.loads(need_convert.lower())
+
+    insert_resource_data(
+        folder=folder,
+        resource=resource,
+        data_source=datasource,
+        from_json=False,
+        need_convert=need_convert,
+    )
+    search_omero_app.logger.info("Indexing data in progress...")
+
+    import time
+
+    time.sleep(60)
+    if update_cache:
+        update_data_source_cache(datasource)
+
+
+@manager.command
+@manager.option(
+    "-f",
+    "--file_name",
+    help="CSV file name",  # noqa
+)
+@manager.option(
+    "-r",
+    "--resource",
+    help="resource name, creating all the indexes for all the resources is the default",  # noqa
+)
+def convert_to_searchengine_indexer_format(file_name=None, resource=None):
+    from omero_search_engine.cache_functions.elasticsearch.transform_data import (
+        conver_to_searchengine_fromat,

```

Comment:
Typo in the name it should be ``convert_to_searchengine_format``.
Please update accordingly

Reply:
Could you fix it please?

Reply:
It has been fixed.